pkg/audit/verifier.go

```diff
@@ -489,24 +517,45 @@ func (verifier *Verifier) GetShare(ctx context.Context, limit *pb.AddressedOrder
 	}, nil
 }
 
+// RemoveFailedPieces removes lost pieces from a pointer
+func (verifier *Verifier) RemoveFailedPieces(ctx context.Context, path string, pointer *pb.Pointer, failedPieces storj.NodeIDList) (err error) {
```

Comment:
nit - `failedPieces` should be called `failedPieceNodeIDs`

Reply:
fixing